src/data.ml

```diff
@@ -1279,7 +1283,11 @@ let pp_tab_args fmt l =
 
 let pp_arg sep fmt (dir,ty,doc) =
   let dir = if dir then "i" else "o" in
-  Fmt.fprintf fmt "%s:%s%s" dir ty sep
+  try 
```

Comment:
No more useful: the change aims to put parentheses around arrow types.
This was needed in a previous version of the compiler, but no more in the current one